tony-core/src/main/java/com/linkedin/tony/util/Utils.java

```diff
@@ -618,5 +618,50 @@ public static void extractResources() {
     }
   }
 
+  /**
+   * Parses Docker related configs and sets the appropriate container environment variables if Docker is available.
+   * Uses reflection to support older versions of Hadoop.
+   */
+  public static boolean parseDockerConfigs(Configuration tonyConf, Map<String, String> containerEnv, String jobType) {
```

Comment:
Please explain in the Javadoc what the boolean returned means.

Reply:
Can we also rename this method something like `setDockerEnvironmentVariables`? `parseXXX` doesn't sound like it has side effects, but this method does have side effects.

Reply:
Yah, that makes sense. I'll update the signature of this function